cli/determined_cli/experiment.py

```diff
@@ -202,7 +183,7 @@ def local_experiment(args: Namespace) -> None:
     #
     # Reference: https://docs.python.org/3/library/sys.html#sys.path
 
-    with experimental._local_execution_manager(args.model_def.resolve()):
+    with det._local_execution_manager(args.model_def.resolve()):
         trial_class = load.load_trial_implementation(experiment_config["entrypoint"])
         sys.path = [""] + sys.path
```

Comment:
Not sure how or when this happened but this sys.path edit looks like it is 1) a duplicate, and 2) not necessary anymore.